src/stylus/elements/_global.styl

```diff
@@ -11,6 +11,8 @@ html
   -moz-osx-font-smoothing: grayscale
   -webkit-tap-highlight-color: rgba(0, 0, 0, 0)
   overflow-x: hidden
+  &.ovf-hidden
```

Comment:
add this to src/stylus/trumps/_display.styl as helper classes for **.overflow-hidden**, **.overflow-x-hidden** and **overflow-y-hidden**.